x/ccv/provider/keeper/distribution.go

```diff
@@ -272,7 +282,17 @@ func (k Keeper) GetConsumerRewardsPool(ctx sdk.Context) sdk.Coins {
 // for the given consumer chain
 func (k Keeper) ComputeConsumerTotalVotingPower(ctx sdk.Context, chainID string) (totalPower int64) {
 	// sum the consumer validators set voting powers
-	for _, v := range k.GetConsumerValSet(ctx, chainID) {
+	vals, err := k.GetConsumerValSet(ctx, chainID)
```

Comment:
Why not call this `consumerVals` instead of `vals` as you do above in `AllocateTokensToConsumerValidators`?

Reply:
because it's clear from the context of being in ComputeConsumerTotalVotingPower, and the function is not so large that it's easy to forget